test/module/irohad/simulator/simulator_test.cpp

```diff
@@ -84,10 +85,25 @@ TEST_F(SimulatorTest, ValidWhenInitialized) {
 
 TEST_F(SimulatorTest, ValidWhenPreviousBlock) {
   // proposal with height 2 => height 1 block present => new block generated
-  auto txs = std::vector<model::Transaction>(2);
-  auto proposal = model::Proposal(txs);
-  proposal.height = 2;
-
+  auto tx = shared_model::proto::TransactionBuilder()
+                .txCounter(2)
+                .createdTime(iroha::time::now())
+                .creatorAccountId("admin@ru")
+                .addAssetQuantity("admin@tu", "coin#coin", "1.0")
+                .build()
+                .signAndAddSignature(
+                    shared_model::crypto::DefaultCryptoAlgorithmType::
+                        generateKeypair());
+  std::vector<shared_model::proto::Transaction> txs = {tx, tx};
+  auto proposal_p =
+      std::unique_ptr<model::Proposal>(shared_model::proto::ProposalBuilder()
+                                           .height(2)
+                                           .createdTime(iroha::time::now())
+                                           .transactions(txs)
+                                           .build()
+                                           .makeOldModel());
+  auto proposal = *proposal_p;
+  proposal.created_time = iroha::time::now();
```

Comment:
What's the point? It's set at the 101 line, isn't it?

Reply:
no:( makeOldModel doesn't do it's bug (or feature)